boards/slstk3402a/include/board.h

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2015-2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_slstk3402a Silicon Labs SLSTK3402A starter kit
+ * @ingroup     boards
+ * @brief       Support for the Silicon Labs SLSTK3402A starter kit
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the SLSTK3402A starter kit
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Bas Stottelaar <basstottelaar@gmail.com>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "periph_conf.h"
+#include "periph/gpio.h"
+#include "periph/spi.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Xtimer configuration
+ *
+ * The timer runs at 250 KHz to increase accuracy.
+ * @{
+ */
+#define XTIMER_HZ           (250000UL)
+#define XTIMER_WIDTH        (16)
```

Comment:
I don't know if all boards are configured like this, but it looks to me a bit weird. Isn't possible to run the timer at 1MHz? 250KHz is a scaled frequency so you might run in the troubles as in #9049. Can you verify?

Reply:
Other boards are the same. I changed to 250kHz because I did not have enough accuracy to play with. It uses two 16-bit timers in cascading mode, although this chip (generation) has support for a 32-bit timer that I will add support for.